salesagility/SuiteCRM
# Fix #9475 - Update GitHub Templates

## Description




Update GitHub Templates


## Motivation and Context


## How To Test This


1. Pull down changes
2. See that the changes have taken effect

## Types of changes


### Final checklist
